tmpl/cookie-consent-policy.php

```diff
@@ -5,12 +5,13 @@
  * @package Altis-Consent
  */
 
-$options     = get_option( 'cookie_consent_options' );
-$policy_page = $options['policy_page'];
+use Altis\Consent\Settings;
+
+$policy_page = (int) Settings\get_consent_option( 'policy_page' );
 ?>
 
 <div class="cookie-consent-policy">
-	<a href="<?php echo esc_url( get_permalink( (int) $policy_page ) ); ?>">
+	<a href="<?php echo esc_url( get_permalink( $policy_page ) ); ?>">
```

Comment:
What happens when a `policy_page` isn't set by the user? Does this become `get_permalink( 0 );` and is that intentional?

Reply:
Good point, @jazzsequence you could return early before breaking out of PHP if the value is falsy.

Reply:
Hmm...well, this makes me think two things...
1) this should actually use [`get_cookie_policy_url`](https://github.com/humanmade/altis-consent/wiki/Altis-Consent-Public-Functions#get_cookie_policy_url) which is literally _for_ this purpose and...
2) that function should do whatever [`get_privacy_policy_url`](http://developer.wordpress.org/reference/functions/get_privacy_policy_url/) does (which doesn't necessarily disqualify bailing early if there's no URL set).
🤔

Thanks, gentlemen. 🙏